docs/HowToArchiveJournal.md

```diff
@@ -17,13 +17,134 @@ For more info on NuoDB database journaling, please consult the official [NuoDB d
 
 Since the `journal` has to write all commits durably to disk, the speed of the disk directly influences the commit latency.
 To achieve the best performance, NuoDB recommends placing the `journal` on the fastest disk available.
+In contract, the `archive` should be using the disk with the largest throughput.
 
```

Comment:
Line 14, suggest

For more info on NuoDB database journaling, please **refer** the official [NuoDB **Documentation**](https://doc.nuodb.com/nuodb/latest/database-administration/about-database-journaling/).